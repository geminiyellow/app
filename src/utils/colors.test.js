describe('colors', () => {
  it('are as expected', () => {
    expect(require('./colors')).toMatchSnapshot()
  })
})
