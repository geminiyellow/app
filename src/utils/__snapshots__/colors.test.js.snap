// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors are as expected 1`] = `
Object {
  "colorBlack": "#000",
  "colorGray": "#9a9a9a",
  "colorGrayDark": "#777",
  "colorGrayDarker": "#444",
  "colorGrayDarkest": "#222",
  "colorGrayLight": "#ccc",
  "colorGrayLighter": "#eee",
  "colorGrayLightest": "#f2f2f2",
  "colorGreen": "#5CB22B",
  "colorOrange": "#fd7400",
  "colorPurple": "#b008b2",
  "colorPurpleDark": "#650566",
  "colorPurpleDarker": "#4B004C",
  "colorPurpleLight": "#FC0BFF",
  "colorRed": "#fd4100",
  "colorTransparent": "transparent",
  "colorWhite": "#fff",
  "colorYellowLight": "#ffffaa",
  "darken": [Function],
  "lighten": [Function],
  "toRGBA": [Function],
}
`;
